### DISCUSSION TITLE
Microsoft Edge Webview2 Runtime 108.0.1462.42 failing on install

### DISCUSSION THREAD

**Discussion by JimmyTheExploder** [MicrosoftEdge/WebView2Feedback#4005] [Q&A]
Recently at work I deployed updates for Microsoft Edge and the Edge Webview2 Runtime (both version  108.0.1462.42). We use SCCM and WSUS to deploy our security updates. Workstations are running Windows 10, 21H2.

I'm seeing errors with the install on about half our devices. On devices where the update failed to install, WUAHandler.log indicates the update is missing.

When I look at WindowsUpdate.log and find where the install appears to happen, I see:
Deployment job Id [ID NUMBER HERE] : Ending Installing updates, CallerId = CcmExec, Exit code = 0x00000000

But then next line is:
*FAILED* [800700B7] Handler process got unexpected result: 0, exiting

Haven't ever had issues w/ the Webview2 Runtime install until this version.

**Comment by victorthoang** [comment]
Hello @JimmyTheExploder,

Thanks for your inquiry! I've assigned this to a dev that might best answer your question.

**Comment by LiangTheDev** [comment]
The last release of Edge WebView2 Runtime had some build issue and were not released with the same version as Edge browser. Edge browser was released as 108.0.1462.42 and Edge WebView2 Runtime was released two days later as 108.0.1462.44. That's probably why trying to update WebView2 Runtime to 108.0.1462.42 failed.
The next version 108.0.1462.46 is in the process of pushing out for both Edge browser and WebView2 Runtime. That should not have this problem.
